lerna.json

```diff
@@ -11,5 +11,6 @@
         "release/*"
       ]
     }
-  }
+  },
+  "useNx": true
```

Comment:
NX greatly improves the output from tests and allows us to better paralyze the unit tests. Note this is already available as it's baked into lerna now I just had to turn it on.